pkg/cable/metrics.go

```diff
@@ -84,6 +84,16 @@ var (
 			connectionsStatusLabel,
 		},
 	)
+	privateConnectionsGauge = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "submariner_connections_private",
+			Help: "Number of connections and corresponding status (by cable driver and status)",
```

Comment:
It might be helpful to use a different help text for this and `submariner_connections`, and maybe explain more what "private" means in this context and the difference in how the two should be used.